src/collections/resources/tutorials/functional-testing-with-cypress-in-meshery-ui/index.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: "Functional Testing with Cypress in Meshery UI"
```

Comment:
This is a blog post right - https://layer5.io/blog/meshery/functional-testing-with-cypress-in-meshery-ui
We can remove this from tutorials.